packages/components/toast/stories/toast.stories.tsx

```diff
@@ -13,6 +13,7 @@ import {
 import {Drawer, DrawerContent} from "@heroui/drawer";
 
 import {Toast, ToastProps, ToastProvider, addToast, closeAll} from "../src";
+import {closeToast} from "../src/toast-provider";
```

Comment:
I think `closeToast` should be exported in `toast/src/index.ts` if it is not for internal usage. Also please also update docs.